src/test/java/net/mcreator/integration/ui/DialogsTest.java

```diff
@@ -209,6 +205,30 @@ public class DialogsTest {
 				false));
 	}
 
+	@Test public void testUsagesSearchDialogs() throws Throwable {
+		UITestUtil.waitUntilWindowIsOpen(mcreator,
+				() -> SearchUsagesDialog.show(mcreator, L10N.t("dialog.search_usages.type.mod_element"),

```

Comment:
Although I see there is one other case of this, it is unusual to test keys here as it does not guarantee that the same key will be used outside tests. You can reduce this to a single execution of the dialog with "" for the message here I would say.